.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @NVIDIA/container-canary-codeowners
```

Comment:
@jacobtomlinson I think you'll need to give this team write access to this repo (by adding it as a collaborator in the repo settings).

<img width="1148" alt="image" src="https://github.com/NVIDIA/container-canary/assets/7608904/7a4d823b-dfc5-484a-91f7-d630ad717ba3">

Reply:
Done, thanks @jameslamb